x-pack/legacy/plugins/actions/server/builtin_action_types/lib/string_utils.ts

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+// see: https://en.wikipedia.org/wiki/Unicode_control_characters
+const CONTROL_CHAR_PATTERN = /[\x00-\x1F]|[\x7F-\x9F]|[\u2028-\u2029]/g;
```

Comment:
I think this will remove tabs as well?  I think that is probably OK, but some might expect that to be valid  whitespace.

Reply:
true; it's likely to confuse customers, so I think we should probably just let that one pass through untranslated